keyboards/foxlab/key65/universal/info.json

```
@@ -0,0 +1,230 @@
+                {"label":"K11 (D1,F5)", "x":1.5, "y":1},
+                {"label":"K12 (D1,F6)", "x":2.5, "y":1},
+                {"label":"K13 (D1,F7)", "x":3.5, "y":1},
+                {"label":"K14 (D1,C7)", "x":4.5, "y":1},
+                {"label":"K15 (D1,C6)", "x":5.5, "y":1},
+                {"label":"K16 (D1,B6)", "x":6.5, "y":1},
+                {"label":"K17 (D1,B5)", "x":7.5, "y":1},
+                {"label":"K18 (D1,B4)", "x":8.5, "y":1},
+                {"label":"K19 (D1,D7)", "x":9.5, "y":1},
+                {"label":"K1A (D1,D6)", "x":10.5, "y":1},
+                {"label":"K1B (D1,D4)", "x":11.5, "y":1},
+                {"label":"K1C (D1,D5)", "x":12.5, "y":1},
+                {"label":"K1D (D1,D3)", "x":13.5, "y":1, "w":1.5},
+                {"label":"K1F (D1,B0)", "x":15, "y":1},
+                {"label":"K20 (F0,B1)", "x":0, "y":2, "w":1.75},
+                {"label":"K21 (F0,F5)", "x":1.75, "y":2},
+                {"label":"K22 (F0,F6)", "x":2.75, "y":2},
+                {"label":"K23 (F0,F7)", "x":3.75, "y":2},
+                {"label":"K24 (F0,C7)", "x":4.75, "y":2},
+                {"label":"K25 (F0,C6)", "x":5.75, "y":2},
+                {"label":"K26 (F0,B6)", "x":6.75, "y":2},
+                {"label":"K27 (F0,B5)", "x":7.75, "y":2},
+                {"label":"K28 (F0,B4)", "x":8.75, "y":2},
+                {"label":"K29 (F0,D7)", "x":9.75, "y":2},
+                {"label":"K2A (F0,D6)", "x":10.75, "y":2},
+                {"label":"K2B (F0,D4)", "x":11.75, "y":2},
+                {"label":"K2D (F0,D3)", "x":12.75, "y":2, "w":2.25},
+                {"label":"K2F (F0,B0)", "x":15, "y":2},
+                {"label":"K30 (F4,B1)", "x":0, "y":3, "w":1.25},
+                {"label":"K31 (F4,F5)", "x":1.25, "y":3},
+                {"label":"K32 (F4,F6)", "x":2.25, "y":3},
+                {"label":"K33 (F4,F7)", "x":3.25, "y":3},
+                {"label":"K34 (F4,C7)", "x":4.25, "y":3},
+                {"label":"K35 (F4,C6)", "x":5.25, "y":3},
+                {"label":"K36 (F4,B6)", "x":6.25, "y":3},
+                {"label":"K37 (F4,B5)", "x":7.25, "y":3},
+                {"label":"K38 (F4,B4)", "x":8.25, "y":3},
+                {"label":"K39 (F4,D7)", "x":9.25, "y":3},
+                {"label":"K3A (F4,D6)", "x":10.25, "y":3},
+                {"label":"K3B (F4,D4)", "x":11.25, "y":3},
+                {"label":"K3C (F4,D5)", "x":12.25, "y":3, "w":1.75},
+                {"label":"K3D (F4,D3)", "x":14, "y":3},
+                {"label":"K3F (F4,B0)", "x":15, "y":3},
+                {"label":"K40 (F1,B1)", "x":0, "y":4, "w":1.25},
+                {"label":"K41 (F1,F5)", "x":1.25, "y":4, "w":1.25},
+                {"label":"K42 (F1,F6)", "x":2.5, "y":4, "w":1.25},
+                {"label":"K44 (F1,C7)", "x":3.75, "y":4, "w":2.75},
+                {"label":"K46 (F1,B6)", "x":6.5, "y":4, "w":1.25},
+                {"label":"K48 (F1,B4)", "x":7.75, "y":4, "w":2.25},
```

The longer key is actually to the right here.

```suggestion
                {"label":"K44 (F1,C7)", "x":3.75, "y":4, "w":2.25},
                {"label":"K46 (F1,B6)", "x":6, "y":4, "w":1.25},
                {"label":"K48 (F1,B4)", "x":7.25, "y":4, "w":2.75},
```

---

Good catch